### ISSUE TITLE
improve compass angle change

### ISSUE THREAD

**Issue by jotpe** [mapillary/mapillary_issues#2268]
Entering the number of degree of the compass angle is not easy when you have to do it a lot.
I think this GeoLocator tool has a nice feature. If you press shift you see the direction (red line), even while you are zooming. What do you think?

![bildschirmfoto von 2016-09-16 20 37 04](https://cloud.githubusercontent.com/assets/3539168/18597310/acc56788-7c4d-11e6-9d7d-0f903e399dbe.png)

**Comment by govvin** [comment]
I agree.  A more basic improvement, like adding a spinner to change the compass angle values could be helpful, too.

![image](https://cloud.githubusercontent.com/assets/5821688/19099933/526fb71c-8aed-11e6-8894-498087924bb8.png)

**Comment by angelday** [comment]
Visual compass angle editing is on our todo list, thanks for reinforcing the importance. If you have more thoughts regarding this or you feel like you wanted to share improvements regarding workflow let us know here.

**Comment by mschwendener** [comment]
> improvements regarding workflow 

In the web interface, let me change the direction of the angle of view by dragging it around the dot with the mouse.

**Comment by jotpe** [comment]
Is this no strong need?

**Comment by angelday** [comment]
Keeping it as feature request and moving over to the forum.

_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the Mapillary Forum in the coming weeks. We’ll notify here once it’s been done. See you there!_
